src/features/expandos/index.ts

```diff
@@ -92,37 +127,81 @@ export default class Expandos extends OLFeature {
         return null;
     }
 
-    private async createGallery(
-        expandoProvider: ExpandoProvider,
-        post: HTMLDivElement
-    ) {
-        const imgLinks = await expandoProvider.createGalleryData(post);
-        const gallery = document.createElement("div");
-        for (const [imgLink, imgDesc] of imgLinks) {
+    private createGallery(galleryEntries: GalleryEntryData[], postId: string) {
+        const lg = this.createLightGallery(galleryEntries);
+        const gallery = {
+            data: { id: postId, entries: galleryEntries },
+            lightGallery: lg,
+        };
+        this.galleries[postId] = gallery;
+        return gallery;
+    }
+
+    private createLightGallery(galleryEntries: GalleryEntryData[]) {
+        const galleryDiv = document.createElement("div");
+        for (const { imageSrc, caption } of galleryEntries) {
             const imageAnchorEl = document.createElement("a");
-            if (expandoProvider.usesDataSet) {
-                imageAnchorEl.dataset.src = imgLink
-                imageAnchorEl.dataset.lgSize = "1280-720"
```

Comment:
I couldn't tell in testing what difference using these makes instead of just setting `imageAnchorEl.href = imgLink`. The youtube videos seem to appear at the same size regardless?

Reply:
I follow the guidelines from lightGallery website for inserting youtube videos. I guess they also have compatibility for that. Too bad I can't seem to figure out how to make v.redd.it work, it uses DASH, and videojs works just fine with standalone, but not when inside lgVideo running from content-script.

Reply:
I couldn't get v.redd.it to work even outside of the lightgallery, do you have a branch with the working video.js standalone?

Reply:
I just injected a div of videojs next to header and ran it videojs on that element. Dont have the code cause I removed the stash by accident

Reply:
Oh nice I see you managed to get it working!